tmpvendor/knative.dev/networking/test/test_images/grpc-ping/main.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
```

Comment:
Hi @fhuzero do we still need this directory

Reply:
In theory, this directory can be replaced with its online version hosted on GitHub. I've been trying to replace it and reorg Knative-related test directory but encountered some bugs. I'll remove the directory once reorg is done.

Reply:
Got it. Thanks for the explanation!